tests/track/platform_trackers/test_mlflow_tracker.py

```diff
@@ -0,0 +1,206 @@
+    log_metric("foo", random() + 1)
+    log_metric("foo", random() + 2)
+    # Create an artifact and log it
+    test_directory = tempfile.TemporaryDirectory()
+    with open(f"{test_directory.name}/test.txt", "w") as f:
+        f.write("hello world!")
+    log_artifacts(test_directory.name)
+    test_directory.cleanup()
+
+
+def lgb_run(context):
+    mlflow.set_tracking_uri(context.artifact_path)
+
+    # prepare train and test data
+    iris = datasets.load_iris()
+    X = iris.data
+    y = iris.target
+    X_train, X_test, y_train, y_test = train_test_split(
+        X, y, test_size=0.2, random_state=42
+    )
+
+    # enable auto logging
+    mlflow.lightgbm.autolog()
+
+    train_set = lgb.Dataset(X_train, label=y_train)
+
+    with mlflow.start_run():
+        # train model
+        params = {
+            "objective": "multiclass",
+            "num_class": 3,
+            "learning_rate": 0.1,
+            "metric": "multi_logloss",
+            "colsample_bytree": 1.0,
+            "subsample": 1.0,
+            "seed": 42,
+        }
+        model = lgb.train(
+            params,
+            train_set,
+            num_boost_round=10,
+            valid_sets=[train_set],
+            valid_names=["train"],
+        )
+
+        # evaluate model
+        y_proba = model.predict(X_test)
+        y_pred = y_proba.argmax(axis=1)
+        loss = log_loss(y_test, y_proba)
+        acc = accuracy_score(y_test, y_pred)
+
+        # log metrics
+        mlflow.log_metrics({"log_loss": loss, "accuracy": acc})
+
+
+def xgb_run(context):
+    mlflow.set_tracking_uri(context.artifact_path)
+    # prepare train and test data
+    iris = datasets.load_iris()
+    X = iris.data
+    y = iris.target
+    X_train, X_test, y_train, y_test = train_test_split(
+        X, y, test_size=0.2, random_state=42
+    )
+
+    # enable auto logging
+    mlflow.xgboost.autolog()
+
+    dtrain = xgb.DMatrix(X_train, label=y_train)
+    dtest = xgb.DMatrix(X_test, label=y_test)
+
+    with mlflow.start_run():
+        # train model
+        params = {
+            "objective": "multi:softprob",
+            "num_class": 3,
+            "learning_rate": 0.3,
+            "eval_metric": "mlogloss",
+            "colsample_bytree": 1.0,
+            "subsample": 1.0,
+            "seed": 42,
+        }
+        model = xgb.train(params, dtrain, evals=[(dtrain, "train")])
+        # evaluate model
+        y_proba = model.predict(dtest)
+        y_pred = y_proba.argmax(axis=1)
+        loss = log_loss(y_test, y_proba)
+        acc = accuracy_score(y_test, y_pred)
+        # log metrics
+        mlflow.log_metrics({"log_loss": loss, "accuracy": acc})
+
+
+def test_is_enabled(rundb_mock):
+    # enable tracking in config for inspection
+    mlrun.mlconf.external_platform_tracking.enabled = True
+    # see if mlflow is in scope
+    try:
+        importlib.import_module("mlflow")
+        relevant = True
+    except Exception:
+        relevant = False
```

Comment:
when is this not relevant? mlflow is always installed in tests
test once with mode enabled and once with disabled

Reply:
the try except is not needed because the import will always work. just remove the try except and import part